### PR TITLE
Clean up the logger output #139

### LINKED ISSUE #139: Logger: separate display information and stack trace
@AdityaA1998 try to clean up the logger further.

### PR DESCRIPTION
**Pull request by AdityaA1998**
Fixes #139 

Proposed message:
```
Currently, the logger shows a lot of extra information in console which might 
not be useful for the user.

Let's configure the logger to only show the useful information to the user. 
```

### CODE REVIEW

**Comment by AdityaA1998** [comment]
@eugenepeh @yamidark @yong24s @ongspxm for review

File: src/main/java/reposense/system/CustomLogFormatter.java [hunk 1]
```diff
@@ -13,22 +11,21 @@
...
-        builder.append(formatMessage(record));
-        builder.append("\n");
 
         Throwable t = record.getThrown();
 
         if (t != null) {
-            StringWriter sw = new StringWriter();
-            t.printStackTrace(new PrintWriter(sw));
-            builder.append(sw);
+            builder.append("An error has occurred");
```
Comment (eugenepeh) [inline src/main/java/reposense/system/CustomLogFormatter.java]: This information is not helpful. Let's just show the message to the user in-regardless.
For the stacktrace, let's move to the log file output.

Comment (AdityaA1998) [reply src/main/java/reposense/system/CustomLogFormatter.java]: Stacktrace is already visible in the log file output. This one is just to configure the format of console output

**Comment by AdityaA1998** [comment]
@damithc prof for review

**Pull request merged**